enclave_wrangler/objects_api.py

```diff
@@ -182,33 +185,10 @@ def get_ontologies() -> Union[List, Dict]:
     response_json = response.json()
     return response_json
 
-
+@deprecated(details="Use get_link_types() instead")
 def link_types() -> List[Dict]:
```

Comment:
## Deprecated `link_types()`
I checked and this was doing the same thing (only w/ less options) as `get_link_types()`.